src/main/java/org/maxgamer/quickshop/util/MsgUtil.java

```diff
@@ -497,7 +498,7 @@ public static void send(@NotNull UUID player, @NotNull String message, boolean i
             if (p.getPlayer() != null) {
                 if (msgData.length == 3) {
                     try {
-                        sendItemholochat(p.getPlayer(), msgData[0], Objects.requireNonNull(Util.deserialize(msgData[1])), msgData[2]);
+                        plugin.getQuickChat().sendItemHologramChat(p.getPlayer(), msgData[0], Objects.requireNonNull(Util.deserialize(msgData[1])), msgData[2]);
```

Comment:
Caching quickchat instance may better for performance

Reply:
QuickChat instance already caching in QuickShop.java